### DISCUSSION TITLE
APSIM-Sunflower in NextGen?

### DISCUSSION THREAD

**Discussion by BrianCollinss** [APSIMInitiative/ApsimX#9402] [Q&A] [answered]
Is there any plan to bring APSIM-Sunflower (https://doi.org/10.1016/j.fcr.2016.04.013) to NextGen?

**Comment by hol353** [comment, marked as answer]
There are currently no plans to add a sunflower model to NextGen. It would require funding and some detailed datasets before it can happen.

**Reply by BrianCollinss** [reply]
Cheers.
